doc/users/fonts.rst

```diff
@@ -0,0 +1,120 @@
+All about Matplotlib and Fonts!
+===============================
```

Comment:
```suggestion
Fonts in the Matplotlib text engine
=========================

Users can configure the fonts used by Matplotlib to render text with the default engine.  Users
using TeX to render fonts (``usetex=True``) should consult `APPROPRIATE REFERENCE HERE`.

```

Reply:
Yep, added a slightly modified version of this 👍🏼